examples/README.md

```diff
@@ -2,4 +2,35 @@
 
 Examples are typically generated within a FHIR Implementation Guide. However, the [FHIR Publisher](https://confluence.hl7.org/display/FHIR/IG+Publisher+Documentation) responsible for building Implementation Guides places additional requirements on examples that are incompatible with this IG's [data minimization criteria](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/conformance.html#data-minimization), such as requiring `id` elements and not recognizing absolute URIs like `resource:0` for within-Bundle resolution.
 
-Until these issues with the IG Publisher are resolved, we are storing our example resources in this folder outside the IG build process, and linking from the IG to these files on GitHub.
\ No newline at end of file
+Until these issues with the IG Publisher are resolved, we are storing our example resources in this folder outside the IG build process, and linking from the IG to these files on GitHub.
```

Comment:
Links for the new examples should be added to the examples page for the publisher to pick up.
The examples are listed at /input/data/examples.json.

Reply:
This looks good - a couple of notes.

- [ ] there is a typo in StructureDefinition-vaccination-credential-immunization-resource-examples.html 'foriegn'
- [ ] the Immunization profiles examples list should omit 'dose 1' or 'dose 2' descriptors. The immunization profiles do not include any dose number indicators, that has to be computed from the full bundle. For examples of individual immunization resources, we should leave out dose number from the description. You just need once example for each type.